stdlib/http/src/main/ballerina/src/http/client_endpoint.bal

```diff
@@ -160,14 +160,14 @@ public type Client client object {
     }
 
     # Submits an HTTP request to a service with the specified HTTP verb.
-    # The `Client.submit()` function does not give out a `Response` as the result,
-    # rather it returns an `HttpFuture` which can be used to do further interactions with the endpoint.
+    # The `Client.submit()` function does not give out a `Response` as the result.
```

Comment:
Since this is a remote function, we cannot use dot operation right? Someone must have mistakenly added the dot operation. Shall we fix it too?
We can remove `Client.submit()` or replace it with `Client->submit()`. Need to fix above methods as well